### PR TITLE
Fixed grammatical errors

### PR DESCRIPTION
**Pull request by imshashikantdev**

### CODE REVIEW

File: phabricator-user.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ at the top of any page.
    :alt: Screenshot of Phabricator login button
 
 You'll be taken to another page with a single button, which will in
-turn take you to `bugzilla.mozilla.org
+the turn takes you to `bugzilla.mozilla.org
```
Comment (globau) [inline phabricator-user.rst]: This isn't grammatically better than the original - "the" doesn't belong here.  Drop this change.

File: phabricator-user.rst [hunk 2]
```diff
@@ -57,10 +57,10 @@ account.
    :alt: Screenshot of BMO login button
 
 After you've done so, you'll be redirected back to Phabricator, where
-you will be prompted to create a new Phabricator account.  On this
+you will be prompted to create a new Phabricator account.  In this
 form, the "Real Name" field is taken from your BMO account's real
 name.  If your BMO real name also contains the ``:<ircnick>``
-convention, that is, a username starting with a colon, it will be
+a convention, that is, a username starting with a colon, it will be
```
Comment (globau) [inline phabricator-user.rst]: "a" doesn't belong here; drop this change.

File: phabricator-user.rst [hunk 3]
```diff
@@ -187,8 +186,7 @@ accomplished through the actions dropdown.  The following are
 available to revision authors:
 
 * **Request Review**: Asks the reviewer(s) to take another look at the
-  revision.  If it is not already, the revision status will be changed
-  to "Needs Review".  If a reviewer has previously accepted the
+  revision.  If it is not already, the revision status will be changed to "Needs Review".  If a reviewer has previously accepted the
```
Comment (globau) [inline phabricator-user.rst]: This change just makes the line longer; drop this change.

File: phabricator-user.rst [hunk 4]
```diff
@@ -160,15 +160,14 @@ suggestions for your overall design", etc.).
...
+review later.  Usually, you will want to use the "Add Action..."
 dropdown to signal a clear intent to the revision author and to
 communicate what they should do next.  These actions include:
 
 * **Accept Revision**: The diff is good as it is and can be landed, or
   at most requires small changes that do not need re-review.
 * **Request Changes**: The diff needs some changes before it can be
-  landed.  Specific change requests should be left as comments, as
-  described above.
+  landed.  Specific change requests should be left as comments, as described above.
```
Comment (globau) [inline phabricator-user.rst]: This change just makes the line longer; drop this change.

File: phabricator-user.rst [hunk 5]
```diff
@@ -206,13 +204,9 @@ available to revision authors:
 There is another action available specifically to nonauthors:
 
 * **Commandeer Revision**: Allows you to take over a revision by
-  becoming its author.  Note that the original author will no longer
-  be able to post updated diffs to the revision. Note: Lando doesn't care
-  who owns the revision on Phabricator, but, it does care about the commit
-  author. When updating someone else's commit, you can use
+  becoming its author.  Note that the original author will no longer be able to post updated diffs to the revision. Note: Lando doesn't care who owns the revision on Phabricator, but, it does care about the commit author. When updating someone else's commit, you can use
```
Comment (globau) [inline phabricator-user.rst]: This change just makes the line longer; drop this change.

File: phabricator-user.rst [hunk 6]
```diff
@@ -206,13 +204,9 @@ available to revision authors:
...
   ``hg commit --amend --user "Other Person <example@mozilla.com>"`` or
-  ``git commit --amend --author="Other Person <example@mozilla.com>"`` to set
-  the commit author information to the right person.
+  ``git commits --amend --author="Other Person <example@mozilla.com>"`` to set the commit author information to the right person.
```
Comment (globau) [inline phabricator-user.rst]: This change just makes the line longer; drop this change.

File: phabricator-user.rst [hunk 7]
```diff
@@ -222,9 +216,9 @@ behind the action or other relevant notes.
 Landing Patches
 ***************
 
-For Mercurial repositories, in particular `mozilla-central
+For Mercurial repositories, in particular, `mozilla-central
 <https://hg.mozilla.org/mozilla-central>`_, we highly recommend using
-:doc:`Lando </lando-user>`.  See :ref:`getting-in-touch` to have
+:doc:`Lando </lando-user>`.  See ref:`getting-in-touch` to have
```
Comment (globau) [inline phabricator-user.rst]: RST syntax for references uses `:ref:` not `ref:`; drop this change.

File: phabricator-user.rst [hunk 8]
```diff
@@ -249,11 +243,11 @@ Our Installation
 
 Mozilla's Phabricator instance is a stock installation, with a small patch
 applied, and some custom extensions.  The patch and extensions are
-intentionally small in scope and are limited to supporting integration
+intentionally small in scope and are limited to supporting the integration
```
Comment (globau) [inline phabricator-user.rst]: This reads better with "the"; drop this change.

File: phabricator-user.rst [hunk 9]
```diff
@@ -249,11 +243,11 @@ Our Installation
...
 points with `bugzilla.mozilla.org <https://bugzilla.mozilla.org>`_
 ("BMO").
 
-See :ref:`conduit-repos` for the location of our source code.
+See ref:`conduit-repos` for the location of our source code.
```
Comment (globau) [inline phabricator-user.rst]: Another broken `:ref:` - drop this change

File: phabricator-user.rst [hunk 10]
```diff
@@ -290,7 +284,7 @@ disabled in a Phabricator installation.  However, at the moment
 Mozilla has no defined engineering processes for post-commit review of
 Firefox and related code, so we do not recommend its use, at least
 until such time as a process is deemed necessary and implemented.
-Audit may, of course, be useful to projects hosted on the Mozilla
+An audit may, of course, be useful to projects hosted on the Mozilla
```
Comment (globau) [inline phabricator-user.rst]: "Audit" is the name of a Phabricator app, so it's being used correctly here; drop this change.

File: phabricator-user.rst [hunk 11]
```diff
@@ -324,7 +318,7 @@ Authorization
 
 If a bug has one or more security groups applied to it, that is, it
 has restricted visibility, any Differential revisions associated with
-it are similarly restricted in visibility.  This will initially only
+it are similarly restricted invisibility.  This will initially only
```
Comment (globau) [inline phabricator-user.rst]: The original phrasing is clearer; drop this change

File: phabricator-user.rst [hunk 12]
```diff
@@ -349,9 +343,9 @@ Review flags
 ============
 
 Review flags are not set on Differential stub attachments.  The
-difference in models between the two systems make any such mapping
+the difference in models between the two systems make any such mapping
```
Comment (globau) [inline phabricator-user.rst]: This duplicates the word "the"; drop this change.

File: phabricator-user.rst [hunk 13]
```diff
@@ -349,9 +343,9 @@ Review flags
...
 both difficult and potentially misleading, the requisite information
-is not exposed via Phabricator's Conduit API, and Phacility have
+is not exposed via Phabricator's Conduit API, and Facility have
```
Comment (globau) [inline phabricator-user.rst]: "Phacility" is the correct spelling; drop this change

File: phabricator-user.rst [hunk 14]
```diff
@@ -386,21 +380,16 @@ the main components:
   extensions, we may move them to
   `bugzilla.mozilla.org :: Extensions: PhabBugz
   <https://bugzilla.mozilla.org/enter_bug.cgi?product=bugzilla.mozilla.org&component=Extensions%3A%20PhabBugz>`_
-  depending on where the problem exists in our code.  Also note that, as
-  discussed in :ref:`bmo-integration`, we are strictly limiting
-  customizations to our instance.  We may, however, work with upstream
-  in fixing important issues.
+  depending on where the problem exists in our code.  Also note that, as discussed in ref:`bmo-integration`, we are strictly limiting customizations to our instance.  We may, however, work with upstream in fixing important issues.
```
Comment (globau) [inline phabricator-user.rst]: Broken `:refs:`, and a change that mostly folds multiple lines unnecessarily; drop this change.

File: phabricator-user.rst [hunk 15]
```diff
@@ -386,21 +380,16 @@ the main components:
...
 * `Lando
   <https://bugzilla.mozilla.org/enter_bug.cgi?product=Conduit&component=Lando>`_:
   For issues with Lando, the UI/API for requesting and monitoring commit landings.
 * `Transplant
   <https://bugzilla.mozilla.org/enter_bug.cgi?product=Conduit&component=Transplant>`_:
-  For issues with Transplant, the backend service which takes landing requests from Lando and
-  pushes them to the relevant repository.
+  For issues with Transplant, the backend service which takes landing requests from Lando and pushes them to the relevant repository.
```
Comment (globau) [inline phabricator-user.rst]: This change just makes the line longer; drop this change.

File: phabricator-user.rst [hunk 16]
```diff
@@ -386,21 +380,16 @@ the main components:
...
 * `General
   <https://bugzilla.mozilla.org/enter_bug.cgi?product=Conduit&component=General>`_:
-  Feel free to file issues here if you aren't sure where they should
-  go.  We'll triage them as needed.
+  Feel free to file issues here if you aren't sure where they should go.  We'll triage them as needed.
```
Comment (globau) [inline phabricator-user.rst]: This change just makes the line longer; drop this change.

File: phabricator-user.rst (hunk 1 above)
Comment (imshashikantdev) [reply phabricator-user.rst]: Hi byron jones

"takes" is acceptable in place of "take"?

Comment (globau) [reply phabricator-user.rst]: No, take should be singular here (ie. there's no need to change this sentence).

**Comment by globau** [comment]
Looks like this PR has stall; closing.

**Pull request closed without merging**